testing/run_tests.py

```diff
@@ -1054,6 +1070,19 @@ def main():
         build_dir, engine_filter, args.coverage, args.engine_capture_core_dump
     )
 
+  # Use this type to exclusively run impeller vulkan tests.
```

Comment:
Since this looks like it will be temporary, is there a GitHub issue that can be commented here to track when we might be able to clean it up?

Reply:
done: filed https://github.com/flutter/flutter/issues/113961 and added a ref in the comment